backend/src/services/auth/auth-signup-service.ts

```diff
@@ -358,6 +370,45 @@ export const authSignupServiceFactory = ({
         tx
       );
 
+      if (metadata) {
```

Comment:
May be we can call this something specific rather than a generic metadata. Because later when we read back we may not understand this at all.